File: c/menor_de_tres.c
#include <stdio.h>
#include <stdlib.h>

int main()
{
    int x,y,z;

    printf("Primeiro valor: ");
    scanf("%d", &x);

    printf("Segundo valor: ");
    scanf("%d", &y);

    printf("Terceiro valor: ");
    scanf("%d", &z);

    if (x < y && x < z)
    {
        printf("MENOR VALOR = %d", x);
    }
    else if (y < x && y < z)
    {
        printf("MENOR VALOR = %d", y);
    }
    else
    {
        printf("MENOR VALOR = %d", z);
    }


    return 0;
}

File: csharp/menor_de_tres.cs
using System;
using System.Globalization;

namespace menor_de_tres
{
    internal class Program
    {
        static void Main(string[] args)
        {

            CultureInfo CI = CultureInfo.InvariantCulture;


            int x, y, z;

            Console.Write("Primeiro valor: ");
            x = int.Parse(Console.ReadLine(), CI);

            Console.Write("Segundo valor: ");
            y = int.Parse(Console.ReadLine(), CI);

            Console.Write("Terceiro valor: ");
            z = int.Parse(Console.ReadLine(), CI);

            if (x < y && x < z)
            {
                Console.WriteLine("MENOR = " + x);
            }
            else if (y < z && y < x)
            {
                Console.WriteLine("MENOR = " + y);
            }
            else
            {
                Console.WriteLine("MENOR = " + z);
            }
        }
    }
}
